backend/tests/models/test_category_model.py

```diff
@@ -0,0 +1,30 @@
+from backend.models.category import Category
+
+
+class TestCategory:
+    name = "Category"
+    description = "A very nice category"
+
+    def test_create_instance(self):
+        self.category = Category(self.name, self.description)
+        assert isinstance(self.category, Category)
+        assert self.category.name == self.name
+        assert self.category.description == self.description
+
+    def test_invalid_name(self):
+        invalid_names = ['', 123, 'a' * 1000]
+        for value in invalid_names:
```

Comment:
que inteligente esse for, tu é ninja haha

Reply:
verdade, tb curti bastante, arrasou Ju

Reply:
Bom ponto, @lucasljoliveira-olist.
Creio que como o teste está focado no nome, se falhar já sabemos que foi ali, independente de qual valor inválido foi passado.